tools/nl/embeddings/sheets/dc_nl_svs_curated.csv

```diff
@@ -1,1247 +1,1245 @@
-Amount_Remittance_InwardRemittance,,,,Total Amount of Inward Remittances
-Amount_Remittance_InwardRemittance_AsFractionOf_Amount_EconomicActivity_GrossDomesticProduction_Nominal,,,,Inward Remittances as a Percentage of Nominal Gross Domestic Production
-Amount_Remittance_OutwardRemittance,,,,Total Amount of Outward Remittances
-Amount_Stock,,,,Total Stock Value
-Amount_Stock_AsFractionOf_Amount_EconomicActivity_GrossDomesticProduction_Nominal,,,,Stock Value as a Percentage of Nominal Gross Domestic Production (GDP)
-Count_CycloneEvent,,,,Number of Cyclones
-Count_Death_0Years_AsFractionOf_Count_BirthEvent_LiveBirth,,,,Infant Mortality Rate
-Count_Death_0Years_Female_AsFractionOf_Count_BirthEvent_LiveBirth_Female,,,,Female Infant Mortality Rate
-Count_Death_0Years_Male_AsFractionOf_Count_BirthEvent_LiveBirth_Male,,,,Male Infant Mortality Rate
-Count_Death_DiseasesOfTheCirculatorySystem,,,,Number of Deaths Due to Diseases of the Circulatory System
-Count_Death_DiseasesOfTheNervousSystem,,,,Number of Deaths Due to Diseases of the Nervous System
-Count_Death_DiseasesOfTheRespiratorySystem,,,,Number of Deaths Due to Diseases of the Respiratory System
-Count_Death_ExternalCauses,,,,Number of Deaths Due to External Causes
-Count_Death_Neoplasms,,,,Number of Deaths Due to Neoplasms (Tumors or Abnormal Growths)
-Count_Household_IncomeOf100000To124999USDollar,,,,"Number of Households With an Income Between $100,000 and $124,999"
-Count_Household_IncomeOf10000To14999USDollar,,,,"Number of Households With an Income Between $10,000 and $14,999"
-Count_Household_IncomeOf125000To149999USDollar,,,,"Number of Households With an Income Between $125,000 and $149,999"
-Count_Household_IncomeOf150000To199999USDollar,,,,"Number of Households With an Income Between $150,000 and $199,999"
-Count_Household_IncomeOf200000OrMoreUSDollar,,,,"Number of Households With an Income of $200,000 or More"
-Count_Household_IncomeOf20000To24999USDollar,,,,"Number of Households With an Income Between $20,000 and $24,999"
-Count_Household_IncomeOf40000To44999USDollar,,,,"Number of Households With an Income Between $40,000 and $44,999"
-Count_Household_IncomeOf50000To59999USDollar,,,,"Number of Households With an Income Between $50,000 and $59,999"
-Count_Household_IncomeOf60000To74999USDollar,,,,"Number of Households With an Income Between $60,000 and $74,999"
-Count_Household_IncomeOf75000To99999USDollar,,,,"Number of Households With an Income Between $75,000 and $99,999"
-Count_Household_IncomeOfUpto10000USDollar,,,,"Number of Households With an Income of $10,000 or Less"
-Count_HousingUnit_1940To1949DateBuilt,,,,Number of Housing Units Built Between 1940 and 1949
-Count_HousingUnit_1950To1959DateBuilt,,,,Number of Housing Units Built Between 1950 and 1959
-Count_HousingUnit_1960To1969DateBuilt,,,,Number of Housing Units Built Between 1960 and 1969
-Count_HousingUnit_1970To1979DateBuilt,,,,Number of Housing Units Built Between 1970 and 1979
-Count_HousingUnit_1980To
```

Comment:
Trying to understand the changes in this CSV... these don't seem like just fixes to pass tests, but it seems we are also adding more descriptions?

Reply:
You are right. This has more than just changes to pass tests. Long story is that I realize that in the sheet we've been using (https://docs.google.com/spreadsheets/d/1evJAt0iaPWt5pcw3B7xeAtnp_mneDkUrTL_KMyyS-RQ/) we have more than just one column for Description (there is "Description Strings" and a "Final Description"). We also have another column which was used for some automated alternatives which I had previously excluded but realized there is no need to exclude. So in summary, one switch here is that the "Description" column corresponds to the "Final Description" column in the sheet (instead of "Description Strings"). Second addition is that the "Curated_Alternatives" column now has "Description Strings" column and the left out automated alternatives column from the sheets.

This should bring us to parity with the current embeddings which use ALL of the columns from the sheet.